src/app/components/media/Media.js

```diff
@@ -40,9 +40,11 @@ const MediaContainer = Relay.createContainer(MediaComponent, {
                 text
                 parsed_fragment
                 annotator {
-                  id
-                  name
-                  profile_image
+                  ... on User {
```

Comment:
This is a dangerous pattern. If `annotator` is a `BotUser`, these fields will not exist. In Relay Classic any access will give `undefined`. In Relay Modern, any access will crash (white screen of death).

I searched the code and couldn't find where `annotator` is used. There's one destructuring bit in `MediaTimelineUtils.js` (which would crash, given a `BotUser`) and that's it.

Reply:
Yes, thanks, I think we should ensure `Annotation.annotator` always refers to ` User` (instead of being polymorphic), and that `BotUser` is a superset of `User`.